rustfmt.toml

```diff
@@ -0,0 +1,123 @@
+## General non-formatting configurations
+#
+version = "Two"
+unstable_features = true
+edition = "2018"
+ignore = []
+color = "Auto"
+disable_all_formatting = false
+hide_parse_errors = true
+license_template_path = ""
+report_fixme = "Unnumbered"
+report_todo = "Unnumbered"
+
+## Already agreed upon configurations
+#
+hard_tabs = false
+tab_spaces = 4
+newline_style = "Unix"
+
+## Width configuration
+#
+error_on_unformatted = false
+error_on_line_overflow = false
+use_small_heuristics = "Default"
+max_width = 100
+
+## Import/module styling
+#
+reorder_imports = true
+reorder_modules = true
+imports_indent = "Block"
+imports_layout = "HorizontalVertical"
+merge_imports = true
+skip_children = false
+
+## Match styling
+#
+match_block_trailing_comma = true
+match_arm_blocks = false
+
+## Blank line control
+#
+blank_lines_lower_bound = 0
+blank_lines_upper_bound = 100
+# preserve_block_start_blank_lines = true
+
+## Brace style
+#
+brace_style = "SameLineWhere"
+control_brace_style = "AlwaysSameLine"
+
+## Indent control
+#
+indent_style = "Block"
+
+## Comment control
+#
+wrap_comments = false
+# comment_width = 80
+normalize_comments = false
+
+## Doc comment control
+#
+format_code_in_doc_comments = false
+normalize_doc_attributes = false
+
+## Function formatting
+#
+fn_args_layout = "Tall"
```

Comment:
```suggestion
fn_params_layout = "Tall"
```
Wrong key was used. https://rust-lang.github.io/rustfmt/?version=master&search=#fn_params_layout

Reply:
You'd think so, but apparently they renamed it to `fn_args_layout` in nightly. If you try to run it with your change it complains about an unknown key:
![image](https://user-images.githubusercontent.com/7739085/94308086-d7a75b00-ff2a-11ea-84c1-79521b00fa91.png)

Reply:
Huh, seems like the docs have been wrong, because people have been complaining about this since 1.40 at least.